model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -356,7 +356,8 @@ def _parse_perf_throughput(self, section):
         Parses throughput from the perf analyzer output
         """
 
-        throughput = re.search('Throughput: (\d+\.\d+|\d+)', section)
+        throughput = re.search('Throughput: (\d+\.\d+e\+\d+|\d+\.\d+|\d+)',
```

Comment:
`Throughput:` is a literal string.  Can this be abstracted?  If this string ever changes, two code changes will be needed..

Reply:
@nv-braf is already working on a story that converts this parsing to pull from a CSV instead, which would make this moot (due to column headers).  The most important part of this story is the unit test that will protect the feature when he refactors.